src/OpenTelemetry.Contrib.Instrumentation.AWS/AWSClientInstrumentationOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="AWSClientInstrumentationOptions.cs" company="OpenTelemetry Authors">
```

Comment:
nit: since we are separating everything, probably this should change as well. Each project isn't part of OpenTelemetry Authors IMO, since it will almost be managed by the owners of that project.

Reply:
Right. I guess we would be doing that going forward. I'll update the "company" for this projects file as well as the "Extensions.AWSXRay" in a separate PR together after this one is merged.

Reply:
This repo is owned by OpenTelemetry, so everything in this repo would owned by same "OpenTelemetry authors".
The company needs to change, only if the code is hosted somewhere outside of OpenTelemetry owned repo